docs/sources/getting-started/getting-started-elasticsearch.md

```diff
@@ -0,0 +1,107 @@
++++
+title = "With Grafana and Elasticsearch"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started", "Elastic", "Elasticsearch"]
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+weight = 400
++++
+
+# Getting started with Grafana and Elasticsearch
+
+Elasticsearch is a popular open source search and analytics engine for which Grafana provides out-of-the-box support. This topic walks you through the steps to create your first Elasticsearch backed dashboard in grafana to display any kind of data stored in your Elasticsearch database.
+
+## Step 1. Install Grafana and build your first dashboard
+
+Use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to:
+
+- Install Grafana.
+- Log in to Grafana.
+- Create your first dashboard.
+
+## Step 2. Download and install Elasticsearch
+
+Elasticsearch can be installed on many different operating systems. Refer to the [Installing Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html) for a complete list of all available options.
+
+Alternately you can install Elasticsearch using the resources available in [grafana/grafana](https://github.com/grafana/grafana) GitHub repository (recommended). Here you will find a collection of supported data sources, including Elasticsearch, along with test data and pre-configured dashboards for use.
+
+1. Clone the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository to your local system.
+1. Install Docker or verify that it is installed on your machine.
+1. Within your local `grafana` repository, change directory to [devenv](https://github.com/grafana/grafana/tree/master/devenv).
+   ```
+   cd devenv
+   ```
+1. Run the bash command to setup data sources and dashboards.
+   ```
+   ./setup.sh
+   ```
+1. Restart the Grafana server.
+1. Change directory back to the root directory.
+   ```
+   cd ..
+   ```
+1. Run the make command to create and start Elasticsearch.
+   ```
+    make devenv sources=elastic7
+   ```
+
+This will create and start Elasticsearch, filebeat, metricbeat, kibana and a process that sends random data to your new Elasticsearch instance. We'll ignore filebeat, metricbeat and kibana in this guide.
```

Comment:
Does it start kibana as well? I didn't know that!

Reply:
yup! 🙂